statistics/testdata/trace_suite_in.json

```diff
@@ -0,0 +1,11 @@
+[
+  {
+    "name": "TestTraceCE",
+    "cases": [
+      "a > 0 and a < 2",
```

Comment:
We only add some basic test cases in this PR. More detailed tests will be added afterward.